back/internal/dbg/dbg.go

```diff
@@ -0,0 +1,59 @@
+package dbg
+
+import (
+	"io"
+	"log"
+	"sync/atomic"
+)
+
+// dbgLog is thread-safe.
+type dbgLog struct {
+	log *log.Logger
+	out io.Writer
+}
+
+var atomicV atomic.Value
+var dbg = new()
+
+func new() *dbgLog {
+	var dl dbgLog
+	atomicV.Store(&dl)
+	return &dl
+}
+
+func (dl *dbgLog) println(text string) {
+	dbg.log.Println(text)
+}
+
+func (dl *dbgLog) printf(format string, as ...interface{}) {
+	dbg.log.Printf(format+"\n", as...)
+}
```

Comment:
We had a discussion during check-in where these functions would be named as log and logf, the dbgLog type should be renamed as dbg, the dbg var should be renamed to whatever you want, and the atomic.Value is stored in the dbg type itself in the log field.